src/isar/state_machine/state_machine.py

```diff
@@ -333,6 +340,10 @@ def _mission_paused(self) -> None:
         self.current_mission.status = MissionStatus.Paused
         self.current_task.status = TaskStatus.Paused
         self.current_step.status = StepStatus.NotStarted
+        self.current_mission.error_description = ErrorDescription(
+            full_text="The mission was paused manually by an operator",
+            short_text="an operator paused the mission manually",
+        )
```

Comment:
This isn't really an error and doesn't need a description I would argue

Reply:
It's not an error but it does deserve a description in my opinion. Then it is down to the naming of `ErrorDescription` again.